README.md

```diff
@@ -187,42 +373,52 @@ The [`openssl x509 -req`](https://docs.openssl.org/master/man1/openssl-x509/) co
 | -days 365 | Specifies that the newly-generated certificate expires in 365 days. |
 | -copy_extensions copyall | Copy all extensions, except that subject identifier and authority key identifier extensions. |
 
-## Create certificate chain
+### Create certificate chain
 
 Create certificate chain file PEM with certificate issued by CA and CA Root certificate. For example, with the self-signed certificate:
 
-```
+```shell
 cat kms-signing.crt rootCA.crt > chain.pem
 ```
 
-## Run the application 
+By default, when this command is run from the root of this repository, this will place the certificate chain in the root of this repository (file named `chain.pem`). As a reminder, this `chain.pem` file is a chain of certificates (and not a signing key).
+
+### Run the application
 
 1. Run the application by entering this command:
+
+    ```shell
+    python3 app.py
     ```
-    FLASK_KMS_KEY_ID="$KMS_KEY_ID" FLASK_CERT_CHAIN_PATH="./chain.pem" flask run
-    ```
-    You'll see a response like this:
-    ```
+
+    You'll see a response like this:``
+
+    ```shell
+    Using KMS for signing
+    Running example in dev mode with AWS endpoint: http://localhost.localstack.cloud:4566/
     Using KMS key: cdd59e61-b6fa-4d95-b71f-8d6ae3abc123
-    Using certificate chain: ./chain.pem
-    * Debug mode: off
-    WARNING: This is a development server. Do not use it in a production deployment. 
-    Use a production WSGI server instead.
-    * Running on http://127.0.0.1:5000
-    Press CTRL+C to quit
-    ```
-2. Upload and sign image: In another terminal window, use `curl` to upload an image file (the app works only with JPEGs) and have the app sign it by entering a command like this:
+    Using certificate chain: chain.pem
+    Press CTRL+C to stop the server
+    INFO:waitress:Serving on http://0.0.0.0:5000
     ```
+
+2. Upload and sign image: In another terminal window, use `curl` to upload an image file (the app works only with JPEGs) and have the app sign it by entering a command like this:
+
+    ```shell
     curl -X POST -T "<PATH_TO_JPEG>" -o <SIGNED_FILE_NAME>.jpg 'http://localhost:5000/attach'
     ```
+
     For example:
+
+    ```shell
+    curl -X POST -T ~/Desktop/test.jpeg -o signed.jpeg 'http://localhost:5000/attach'
     ```
-    curl -X POST -T ~/Desktop/test.jpeg -o signed.jpeg 'http://localhost:5000/attach' 
-    ```
+
     In this example, the image with signed Content Credentials is saved to `signed.jpeg`.
 
 If you encounter any issues running the `curl` command, try using `127.0.0.1` instead of `localhost`.
 
 Confirm that the app signed the output image by doing one of these:
-- If you've installed C2PA Tool, run `c2patool <SIGNED_FILE_NAME>.jpg`.
+
+- If you've installed [C2PA Tool](https://github.com/contentauth/c2patool), run `c2patool <SIGNED_FILE_NAME>.jpg`.
```

Comment:
C2PA Tool has been moved to `https://github.com/contentauth/c2pa-rs/tree/main/cli`.

Reply:
Updated.